README.md

```diff
@@ -1,73 +1,67 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
-
-# Project Name
-
-> Description the project.
-
-
-## Built With
-
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
-
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
-
-### Prerequisites
-
-### Setup
-
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
-
-
-
-## Authors
-
-👤 **Author1**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-👤 **Author2**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-## 🤝 Contributing
-
-Contributions, issues, and feature requests are welcome!
-
-Feel free to check the [issues page](../../issues/).
-
-## Show your support
-
-Give a ⭐️ if you like this project!
-
-## Acknowledgments
-
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
-
-## 📝 License
-
-This project is [MIT](./MIT.md) licensed.
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Remote Work
+
+> This is remote conference project, Remote work, also called work from home, work from anywhere, telework, remote job, mobile work, and distance work is an employment arrangement in which employees do not commute to a central place of work, such as an office building, warehouse, or retail store.
+
+> **Note, please click the link to watch the video:-**
+ [video explination](https://www.loom.com/share/ad6aac0159374550bc1dea79f91743c2)
+
+## Built With
+
+- HTML
+- CSS
+- Javascript
+- Git and Github
+- Vscode and linter style
+
+## Live Demo (if available)
+
+[Live Demo Link](https://bahoz-ali.github.io/remote_capstone/)
+

```

Comment:
- [ ] A link to the deployed project should be present in the README. Kindly add the link to the deployed page.

Reply:
**But, the link is already there, and it's working, I think you have not checked the link.
Also, you selected the link !!!!!!**